app/java/net/openid/appauthdemo/PendingIntentStore.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2015 The AppAuth for Android Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.openid.appauthdemo;
+
+import android.app.PendingIntent;
+import android.support.annotation.VisibleForTesting;
+import android.util.Log;
+
+import java.util.HashMap;
+import java.util.Map;
+
+// COPIED FROM APPAUTH LIBRARY SINCE IT IS NON-PUBLIC CLASS;
+// MODIFIED, stores only 1 logout pending intent!
+
+
+/**
+ * Stores {@link PendingIntent} associated with each {@link AuthorizationRequest} made via
+ * {@link AuthorizationService#performAuthorizationRequest}.
+ * The pending intents are read and sent by
+ * the {@link RedirectUriReceiverActivity} when the redirect Uri is received.
+ */
+/* package */ public class PendingIntentStore {
```

Comment:
With the `AuthorizationManagementActivity` pattern added in 0.4.0, PendingIntentStore is no longer needed. I would suggest refactoring `AuthorizationManagementActivity` to be more generally useful for both authorization and logout, rather than duplicating code in this manner.